packages/shared-common/src/index.ts

```diff
@@ -98,7 +98,7 @@ export function getMatchedPositions(code: string, matched: string[], hasVariantG
 
   // highlight for plain classes
   let start = 0
-  code.split(/([\s"'`;<>]|:\(|\)"|\)\s)/g).forEach((i) => {
+  code.split(/([\s"'`;<>*]|:\(|\)"|\)\s)/g).forEach((i) => {
```

Comment:
This is fixing the highlighting of classes if `@unocss-include` is provided in multiline comment syntax:
```css
/* @unocss-include */
@screen at-xs { 
  .class {
    display: flex;
    overflow: hidden;
    @apply flex-col-reverse;
  }
}
```